pkgs/applications/graphics/fondo/default.nix

```diff
@@ -68,7 +68,7 @@ stdenv.mkDerivation rec {
     description = "Find the most beautiful wallpapers for your desktop";
     homepage = "https://github.com/calo001/fondo";
     license = licenses.agpl3Plus;
-    maintainers = with maintainers; [ worldofpeace ];
+    maintainers = with maintainers; [ ];
```

Comment:
I wanna catch this.

Reply:
I think you have sufficient permissions to add commits to this branch of mine, or it could be a later PR of your own if you prefer.

Reply:
I prefer a later PR.